smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -596,6 +658,7 @@ def create_run_policy(
 
     def _start_steps(self) -> None:
```

Comment:
Add docstring to helper functions if thats a standard

Reply:
added